app/javascript/packages/document-capture/components/status-message.tsx

```diff
@@ -7,19 +7,13 @@ export const Status = {
   ERROR: 'ERROR',
   SUCCESS: 'SUCCESS',
 };
+interface StatusMessageProps {
+  status: string;
```

Comment:
We've changed the type of this from `Status` to `string`, which is less strict than we had it. Can we reflect this in TypeScript?